### DISCUSSION TITLE
No Web Store

### DISCUSSION THREAD

**Discussion by HamSandwich01** [ScratchAddons/ScratchAddons#6891] [Help] [answered]
I'm on a school chromebook, so the extension is blocked. Does anyone know a way I may be able to bypass this?

**Comment by Samq64** [comment]
You could try https://scratchaddons.com/docs/getting-started/installing/#downloading-the-zipball

There's also [TurboWarp](https://turbowarp.org) which has some of the editor addons.

**Reply by HamSandwich01** [reply]
but how to i add the file to scratch?

**Reply by Samq64** [reply]
If you go to chrome://extensions in your address bar there should be a developer mode toggle in an upper corner. Toggle it on, click load unpacked and select the zip file. If there is no toggle you can't install it that way.

**Reply by HamSandwich01** [reply]
developer mode is disabled tho

**Reply by BroJac5246** [reply, marked as answer]
> developer mode is disabled tho

You're not going to be able to install it without some serious hacking that we can't help you with, and that I would personally advise against because it's not worth the risk nor the consequences. Your best bet would be to contact your school's or school district's IT department and explain to them why they should allow Scratch Addons, though it is still likely that they'd say no.

**Reply by HamSandwich01** [reply]
i am trying to start a coding club, so that could be a reason why we need it.

**Reply by BroJac5246** [reply]
> i am trying to start a coding club, so that could be a reason why we need it.

Sure, that works. Best of luck!

**Reply by WorldLanguages** [reply]
I think using TurboWarp online is your best bet. Hopefully its website is not blocked.

**Reply by HamSandwich01** [reply]
turbowarp is blocked too :(

**Reply by WorldLanguages** [reply]
Is there a way to download TurboWarp as an html/mhtml file and distribute it over pendrives? 😅 Is humanity going forwards or backwards?
